src/macros.rs

```diff
@@ -43,7 +67,7 @@ macro_rules! jsonrpc_client {
             pub fn polymorphize(self) -> RpcRequest<serde_json::Value> {
                 RpcRequest {
                     method: self.method,
-                    params: serde_json::from_str(&serde_json::to_string(&self.params).unwrap()).unwrap(),
```

Comment:
Need to make it so the params that are optional on the end are not sent, because there are deprecated nodes that can't be sent null and check the optional by the params length.